server/src/main/java/app/friendship/FriendshipService.java

```diff
@@ -51,41 +52,73 @@ public void acceptFriendRequest(Integer friend1Id, Integer friend2Id) {
 
     public void removeFriend(Integer senderId, Integer friendId) {
         friendshipRepository.deleteAll(friendshipRepository.findByStatusAndSenderId(
-            FriendshipStatus.ACCEPTED,
-            senderId
+                FriendshipStatus.ACCEPTED,
+                senderId
         ));
 
         friendshipRepository.deleteAll(friendshipRepository.findByStatusAndSenderId(
-            FriendshipStatus.ACCEPTED,
-            friendId
+                FriendshipStatus.ACCEPTED,
+                friendId
         ));
     }
 
     public List<Integer> getPendingFriendRequests(Integer userId) {
         return friendshipRepository.findByReceiverIdAndStatus(userId, FriendshipStatus.PENDING).stream()
-            .map(Friendship::getSenderId)
-            .toList();
+                .map(Friendship::getSenderId)
+                .toList();
     }
 
+    public List<Integer> getSentFriendRequests(Integer userId) {
+        return friendshipRepository.findByStatusAndSenderId(FriendshipStatus.PENDING, userId).stream()
+                .map(Friendship::getReceiverId)
+                .toList();
+    }
+
+
     public boolean areFriends(Integer senderId, Integer userId) {
         return friendshipRepository.existsBySenderIdAndReceiverIdAndStatus(senderId, userId, FriendshipStatus.ACCEPTED)
-            || friendshipRepository.existsBySenderIdAndReceiverIdAndStatus(userId, senderId, FriendshipStatus.ACCEPTED);
+                || friendshipRepository.existsBySenderIdAndReceiverIdAndStatus(userId, senderId, FriendshipStatus.ACCEPTED);
     }
 
     public List<Integer> getFriends(Integer senderId, Integer userId) {
-        if (!areFriends(senderId, userId)) {
-            return Collections.emptyList();
-        }
+//        if (!areFriends(senderId, userId)) {
+//            return Collections.emptyList();
+//        }
 

```

Comment:
<img width="1041" alt="Screenshot 2024-12-27 at 01 10 25" src="https://github.com/user-attachments/assets/84f940a3-2643-4ae7-92ea-0cbc2ea1e7ac" />

if you are not sing something just remove it

i understand that you remvoed that if because you want to see all friends from all users . i think only my friends can see my friends, but its ok. let remove restrictions for now. later we will create a better service to handle restrictions. ( i removed in in my commit so you dont have to )